src/SqlAsyncCollector.cs

```diff
@@ -542,7 +543,7 @@ AS NewData
             /// <returns>TableInformation object containing primary keys, column types, etc.</returns>
             public static async Task<TableInformation> RetrieveTableInformationAsync(SqlConnection sqlConnection, string fullName, ILogger logger, IEnumerable<string> objectColumnNames)
             {
-                Dictionary<TelemetryPropertyName, string> sqlConnProps = sqlConnection.AsConnectionProps();
+                Dictionary<TelemetryPropertyName, string> sqlConnProps = sqlConnection.AsConnectionProps(EngineEdition.NotQueried.ToString());
```

Comment:
It would be preferable to have this (and any other places that use this "NotQueried" value) to instead have the engine edition and other fields populated once on startup with the initial connection and then added to all events after that.

We may want to look into when/where we send these connection properties at a later point - since right now it seems a bit random (maybe they should even be common properties attached to all events). But for now I'd just say in each of these classes populate it once on the "initial" connection and then use that value as needed in the below places. 

We shouldn't ever send a "NotQueried" value for the engine edition - just don't send it if we don't need/want to add it.